code/modules/mob/mob_grab_specials.dm

```diff
@@ -147,3 +147,46 @@
 
 	if(can_eat)
 		H.attempt_devour(target, H.eat_types, H.mouth_size)
+
+
+/obj/item/weapon/grab/proc/hair_pull(mob/living/carbon/human/target, mob/attacker, var/target_zone)
+
+
+	var/mob/living/carbon/human/H = target

```

Comment:
Why are you doing this? H = target and target = H. These two variables are exactly the same. Normally you would do this if you want to become _more_ precise.